src/main/kotlin/app/revanced/patches/youtube/layout/seekbar/bytecode/patch/SeekbarColorBytecodePatch.kt

```diff
@@ -47,7 +47,7 @@ class SeekbarColorBytecodePatch : BytecodePatch(
             addInstructions(
                 registerIndex + 1,
                 """
-                        invoke-static { v$colorRegister }, $INTEGRATIONS_CLASS_DESCRIPTOR->getSeekbarColorValue(I)I
+                        invoke-static { v$colorRegister }, $INTEGRATIONS_CLASS_DESCRIPTOR->getVideoPlayerSeekbarColorOverride(I)I
```

Comment:
Are we sure to use "Override" as a suffix? Currently, this is not conventionally populated to all similar methods from other patches.